PHPCompatibility/Sniffs/Constants/NewConstantsSniff.php

```diff
@@ -6803,6 +6803,10 @@ class NewConstantsSniff extends Sniff
             '7.4' => true,
         ],
         'T_BAD_CHARACTER' => [
+            '5.6' => true,
+            '7.0' => false,
+            '7.1' => false,
+            '7.2' => false,
```

Comment:
This change breaks the functionality of the `ComplexVersionNewFeatureTrait::getVersionInfo()` method as this array is expected to only have a `false` value _before_ a `true` value.

Reply:
Nothing in the test suite complained about this...